conda-forge.yml

```diff
@@ -1,6 +1,6 @@
-travis:
-  secure:
-    BINSTAR_TOKEN: QrINo3MZOiA7pi1p9M58g3QJauPGTic623f+fVUgHMGBU4Hoap7Uqc+pqc0Wrl2UT/6C/TILcgjVqFO6d7mqX36epg3fUG+TpeSk/82fQN7/K5HfZ/xtG55/R4OvnUdfk8n5cVVtXUoBOexTo8EKa6DEveOUR9Fp0TavFSTorsQyCkJ+yHZzhgXsd5v0ex2RvUtMJWzLXu6PBuKacMWXL0HPI5bjuYab+V4/eMdxWarGSbKcabq02PNFC/VZRnJLWMKDzNKDpwyISoNvZ80h1MMWEHqZDNgQ2gMsWtM1LbjENPe5VVURfeHZRzDshP8E3y4noQFmCAjakH6JRBAim9lG5GV96mOpYRVN1vWWe3rdrEf2YuwcIbxJd6p/0YcPstUavQc61+bAFct1B06pkmNHK19sJo4RcD9WkbmqLKQhu0wfLEMjL+ahCIZ1Y4F/LlfgJjFPuayb8eBLbISwYf6aDGPdkgK6g3dujmEib5s+ofnxKeb6D5puc5xSljdUa0hpb1rklAtPM6g2zBZtH0zW5nlocVPstAMbz2ZMDjHXM5BeQurFEKaGsUf55GyWFnTd8NOAn5jtnT7sMDPEorX7GeGbjgC88/L15123lW0TK6ezALOb71VCuY9t/6UVOicK10Ckqqe002YzgvMwmtQeSTH5JTvtbk3AUNbRkzI=
-appveyor:
-  secure:
-    BINSTAR_TOKEN: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
+travis:
+  secure:
+    BINSTAR_TOKEN: TMTlg8cSMJun5aFyOSGTkuYJSvYpxI92qjBABnIzYJOK/tqeiZNlXNWCzaC++QSJtV/jq332YcupCWcN+BfRdyDzy6A7drDQODV1udMM4ZBg1ggOyUDQ06xakXDKQiDSMXSb0ZJaMcMhtS7yi2JISo1dzwGLTw5HhLDY1kgdPrT3UhGp3e/NloD0L+rsGrpAPbcsupARjFjO1nuaA+8stBuyiyC0f/b3RqqAr5qen2gT6r6N6x7fy+WIugWF+vVj3UgXJhZjxKifOKjASHKYVWA7kV4xraMXXj+ZfWl7a98zQZgCJt3CZq+i/t+SV4UGrDSoYRmrS2tf7I1PAi5g6ZDmycWttbfwTRdCZg9PgBFo8M54UxMKo7ZmXM6MTuU1YWazTcFHbpoFKJdSEhhLEBX5BKA9B6nA2bZXZRkFTXipdEufU6hzJ5+9aoU21JIbLDHHUfscdL8tDK0DPkgjCTWe2ARFXAtnTg+tujKP51Fr4cpgJ2VN8uEsn/BZK+LsvngiaIDD5EAWxfnoHRlOzj/1DTbQvgj0vujv98uFXByVobu9vuAicNZbOaFTpYThCKfqdA9ZBl2XO7sGXdoPuIK5FJrZjwH4MQxKfQUMcBgoBP+OgIXkah71DmJge1Br5tTOgTW3Q3Zr9koJ1T6xerdoSajYwjd6CiVbpohB2ms=

```

Comment:
Why is this value changed? This is why travis-ci didn't upload the conda package